spec/features/add_photo_to_entry_spec.rb

```diff
@@ -20,7 +20,7 @@
     click_link entry.title
     root_photo = "#{Rails.root}/spec/support/images/example_photo.png"
     attach_file "Entry Photo", root_photo
-    click_button 'Add Photo'
+    find("#add-photo",:visible=>false).click
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.